src/components/listitems/ListItemNav.tsx

```diff
@@ -48,15 +52,24 @@ export const ListItemNav = ({
   iconColor = "grey-450",
   paymentLogo,
   accessibilityLabel,
-  testID
+  testID,
+  hideChevron = false,
+  badgeProps
 }: ListItemNav) => {
   const isPressed = useSharedValue(0);
   const { isExperimental } = useIOExperimentalDesign();
   const theme = useIOTheme();
 
-  // TODO: Remove this when legacy look is deprecated https://pagopa.atlassian.net/browse/IOPLT-153
   const listItemNavContent = (
     <>
+      {badgeProps && (
+        <>
+          <View style={{ alignSelf: "flex-start" }}>
+            <Badge {...badgeProps} />
+          </View>
+          <VSpacer size={8} />
```

Comment:
The only change requested:
```suggestion
          <VSpacer size={4} />
```
In the design project we have a distance of `6pt`, but `8pt` is probably too much. Let's use `4pt` so we can make `ListItemNav` more compact.